modules/javafx.graphics/src/main/java/com/sun/javafx/font/WindowsFontMap.java

```diff
@@ -25,103 +25,97 @@
 
 package com.sun.javafx.font;
 
-import java.util.HashMap;
+import java.util.Map;
 
 class WindowsFontMap {
 
-    static class FamilyDescription {
-        public String familyName;
-        public String plainFullName;
-        public String boldFullName;
-        public String italicFullName;
-        public String boldItalicFullName;
-        public String plainFileName;
-        public String boldFileName;
-        public String italicFileName;
-        public String boldItalicFileName;
+    private static class FamilyDescription {
+        private String familyName;
+        private String plainFullName;
+        private String boldFullName;
+        private String italicFullName;
+        private String boldItalicFullName;
+        private String plainFileName;
+        private String boldFileName;
+        private String italicFileName;
+        private String boldItalicFileName;
     }
 
-    static HashMap<String, FamilyDescription> platformFontMap;
+    private static Map<String, FamilyDescription> PLATFORM_FONT_MAP;
```

Comment:
Shouldn't this variable name stay with the normal camel case because it is not final (only static)?

Reply:
Yes, but it's created only once via lazy initialization and then it's effectively final. A matter of taste I would say. There is also a JEP that deals with these cases https://openjdk.java.net/jeps/8209964.